### PR TITLE
Fix manifest list creation

### PR DESCRIPTION
**Pull request by loosebazooka**

### CODE REVIEW

File: cloudbuild_docker.sh [hunk 1]
```diff
@@ -47,6 +47,14 @@ for distro_suffix in "" -debian10 -debian11; do
   docker_manifest gcr.io/$PROJECT_ID/python3${distro_suffix}:debug "amd64 arm64"
 done
 
+# java is only multi-arch from debian11 onwards
+for java_version in -base 11 17; do
+  docker_manifest gcr.io/$PROJECT_ID/java${java_version}-debian11:latest "amd64 arm64"
```
Comment (loosebazooka) [inline cloudbuild_docker.sh:52]: `docker manifest create` -> `docker_manifest`

**Review by priyawadhwa — APPROVED** [review]

**Comment by timja** [comment]
Didn't work from what I can tell?

```
➜  test git:(update-forms) docker run --entrypoint sh -it gcr.io/distroless/java17:debug
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
/ #
```

**Comment by loosebazooka** [comment]
Yeah, so multiplatform images are only published against the images with the debian prefix (consistent with other multiplatform images)

try [gcr.io/distroless/java17-debian11](https://pantheon.corp.google.com/gcr/images/distroless/global/java17-debian11) 

The debug tag points to a manifest list: https://console.cloud.google.com/gcr/images/distroless/global/java17-debian11@sha256:9657209dbc825dee62846f5d06d45234875a4e08297ed7a1dfc26fd46d150e42/details?tab=pull

**Comment by loosebazooka** [comment]
There's probably some space to improve the documentation here.

**Comment by timja** [comment]
Not sure if this is right?

```
➜  jenkins git:(update-forms)  docker run --entrypoint sh -it gcr.io/distroless/java17-debian11:debug
/ # java -version
/busybox/sh: java: not found
```

```
➜  test git:(update-forms) docker run --entrypoint sh -it gcr.io/distroless/java17:debug
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
/ # java -version
openjdk version "17.0.1" 2021-10-19
OpenJDK Runtime Environment (build 17.0.1+12-Debian-1deb11u2)
OpenJDK 64-Bit Server VM (build 17.0.1+12-Debian-1deb11u2, mixed mode, sharing)
```

**Comment by timja** [comment]
Latest image looks ok but not debug:
```
➜  backstage git:(add-timja-to-codeowners) docker run --entrypoint java gcr.io/distroless/java17-debian11:latest -version
openjdk version "17.0.1" 2021-10-19
OpenJDK Runtime Environment (build 17.0.1+12-Debian-1deb11u2)
OpenJDK 64-Bit Server VM (build 17.0.1+12-Debian-1deb11u2, mixed mode, sharing)
➜  backstage git:(add-timja-to-codeowners) docker run --entrypoint java gcr.io/distroless/java17-debian11:debug -version
docker: Error response from daemon: OCI runtime create failed: container_linux.go:380: starting container process caused: exec: "java": executable file not found in $PATH: unknown.
ERRO[0000] error waiting for container: context canceled
```

**Comment by timja** [comment]
submitted https://github.com/GoogleContainerTools/distroless/issues/906

**Pull request merged**